apps/api/src/app/auth/usecases/password-reset-request/password-reset-request.usecase.ts

```diff
@@ -14,16 +14,22 @@ export class PasswordResetRequest {
   private MAX_ATTEMPTS_IN_A_DAY = 15;
   private RATE_LIMIT_IN_SECONDS = 60;
   private RATE_LIMIT_IN_HOURS = 24;
-  constructor(private invalidateCache: InvalidateCacheService, private userRepository: UserRepository) {}
+  constructor(
+    private invalidateCache: InvalidateCacheService,
+    private userRepository: UserRepository,
+    private memberRepository: MemberRepository
+  ) {}
 
   async execute(command: PasswordResetRequestCommand): Promise<{ success: boolean }> {
     const email = normalizeEmail(command.email);
     const foundUser = await this.userRepository.findByEmail(email);
     if (foundUser && foundUser.email) {
       const { error, isBlocked } = this.isRequestBlocked(foundUser);
+      //  const member = await this.memberRepository.findById(foundUser._id);
```

Comment:
Curious if keeping this commented is intended 😸

Reply:
yup haven't implemented it for passwords. cannot find the member just by id. and there is no fn for `findById` in `memberRepository`  we could use others but will need more for it

Reply:
Could you please remove this code if it's not in use in this file? Including the imports part